KHLY2400/KHLY2400-IOC-01App/Db/KHLY2400.db

```diff
@@ -393,8 +482,8 @@ record(bo, "$(P)RES:MODE:SP") {
     field(OUT, "@Keithley2400.proto set_RES_MODE $(PORT)")
     field(ZNAM, "Manual")
     field(ONAM, "Auto")
-	field(VAL, "1")
-	field(PINI, "YES")
+    field(VAL, "1")
+    field(PINI, "YES")
```

Comment:
I don't think the IOC should PINI anything. Please check all of the IOC and ensure it is not sending any special commands on IOC start that could override a user's configuration of the box.